scene/gui/rich_text_label.cpp

```diff
@@ -1943,10 +1943,12 @@ void RichTextLabel::_notification(int p_what) {
 			visible_paragraph_count = 0;
 			visible_line_count = 0;
 
+			//bottom margin for text clipping
+			float v_limit = theme_cache.normal_style->get_margin(SIDE_BOTTOM);
```

Comment:
Follow comment style guide:

```suggestion
			// Bottom margin for text clipping.
			float v_limit = theme_cache.normal_style->get_margin(SIDE_BOTTOM);
```